packages/web/src/pages/profile-page/ProfilePageProvider.tsx

```diff
@@ -1152,6 +1143,25 @@ function mapDispatchToProps(dispatch: Dispatch, props: RouteComponentProps) {
   }
 }
 
+type HookStateProps = {
+  accountUserId?: ID | undefined
+  accountHasTracks?: boolean | undefined
+}
+const hookStateToProps = (Component: typeof ProfilePage) => {
```

Comment:
this is cool, but imo wrapper component might be the move now rather than going hoc route?